### ISSUE TITLE
False positive unused return value Enum.map

### ISSUE THREAD

**Issue by nulian** [rrrene/credo#541]
With credo 0.9.2 and elixir 1.6.3.

Getting a false positive when using Enum.map with `:ets.insert`

Like `:ets.insert(table, Enum.map([1, 2, 3, 4], fn i -> i + 1 end))`

Giving
There should be no unused return values for Enum.map().

**Comment by rrrene** [comment]
Thanks for reporting this 😀  

Is this fixed on `master`?

You can try this by setting the Credo dep to

```elixir
{:credo, github: "rrrene/credo"}
```

Please report back if your issue is solved! 👍

**Comment by rrrene** [comment]
Thanks for reporting this 😀  It is now fixed on `master`.

You can try this by setting the Credo dep to

```elixir
{:credo, github: "rrrene/credo"}
```

Please report back if your issue is solved! 👍
